lib/Index/FileVisibilityChecker.cpp

```diff
@@ -66,10 +66,7 @@ void FileVisibilityChecker::addUnitOutFilePaths(ArrayRef<StringRef> filePaths) {
 
   ReadTransaction reader(DBase);
   for (StringRef filePath : filePaths) {
-    CanonicalFilePath canonPath = CanonPathCache->getCanonicalPath(filePath);
-    if (canonPath.empty())
-      continue;
-    IDCode pathCode = reader.getFilePathCode(canonPath);
+    IDCode pathCode = reader.getUnitPathCode(filePath);
```

Comment:
Could also rename this depending on what the name of `OutFile` becomes. `getUnitFileCode`? That difference seems too subtle to me though.